content/actions/security-guides/security-hardening-for-github-actions.md

```diff
@@ -204,7 +204,7 @@ The same principles described above for using third-party actions also apply to
 
 ## Using OpenSSF Scorecards to secure workflows
 
-[Scorecards](https://github.com/ossf/scorecard) is an automated security tool that flags risky supply chain practices. You can use the [Scorecards action](https://github.com/marketplace/actions/ossf-scorecard-action) and [starter workflow](https://github.com/actions/starter-workflows) to follow best security practices. Once configured, the Scorecards action runs automatically on repository changes, and alerts developers about risky supply chain practices using the built-in code scanning experience. The Scorecards project runs a number of checks, including script injection attacks, token permissions, and pinned actions.
+[OpenSSF Scorecards](https://github.com/ossf/scorecard) automates the risk analysis and trust decisions on the security posture of open source projects and flags risky supply chain practices. You can use the [Scorecards action](https://github.com/marketplace/actions/ossf-scorecard-action) and [starter workflow](https://github.com/actions/starter-workflows) to follow security best practices. Once configured, the Scorecards action runs automatically on repository changes, and alerts developers about risky supply chain practices using the built-in code scanning experience. The Scorecards project runs a number of checks, including script injection attacks, token permissions, and pinned actions.
```

Comment:
I think the GH doc targets repo owners as an audience. `automates the risk analysis and trust decisions on the security posture of open source projects` seems to target consumers. Also, the GH doc is for GH actions.
I know we're trying to be comprehensive, but I'm not sure it's the right place.

I don't have a very strong opinion beyond that, though. If GH are fine adding this change, why not.